README.md

```diff
@@ -702,6 +702,14 @@ const { results } = await Calculator.calculateMeasureReports(measureBundle, pati
 
 **NOTE**: Measure logic highlighting is not available when calculating a population `MeasureReport` with `reportType = 'summary'`
 
+However, highlighted HTML provided by `fqm-execution` is not solely based on whether individual pieces of the measure logic CQL had "truthy" or "falsy" values. When there are populations that depend on each other, a relevance hierarchy is built to ensure that populations that are subsets of others aren't considered for membership until membership is determined for their superset population. If a statement's superset statement is false, then no highlighting will appear, regardless of whether that statement's value is "truthy" or "falsy". This pattern reflects the actual population results that are being returned during calculation.
+
+The following example shows what logic highlighting would look like for a proportion boolean measure where the Initial Population evaluates to false but the Denominator and Numerator statements both evaluated to true.
```

Comment:
I find this line a tiny bit confusing. I wonder if it would be simpler to exclude "but the Denominator and Numerator statements both evaluated to true."? I think that might better explain that the value of these clauses is irrelevant, but def open to conversation here.

Reply:
Definitely open to changing the wording on this, but my reason for adding that the Denominator and Numerator statements in this case evaluate to true was because that is the case where the confusion initially came from. The statement below the screenshot gets into the fact that the value of those clauses are irrelevant, so I could pull that up here. @mgramigna thoughts?

Reply:
Yeah maybe pulling up that statement before the screenshot? Could say something like "the following example shows how the logic highlighting will not style the numerator and denominator despite their truthy raw values bc ... " or something idk